src/components/textarea/textarea.stories.mdx

```diff
@@ -159,6 +159,10 @@ You can use the `required` prop to indicate if the field is mandatory.
 
 #### Customer Border Radius
 
+The `borderRadius` prop accepts a single string that will applied to each corner or an array of values that will be appied in the same order as the CSS border-radius values. 
+
+The array will be applied in the order of: border-top-left-radius, border-top-right-radius, border-bottom-right-radius, border-bottom-left-radius.
+
 <Canvas>
```

Comment:
**suggestion** (non-blocking): I think some examples of custom border-radius values with 2/3/4 individual values (certainly I would include 4) will help make things clearer here (users can inspect the DOM and see what CSS is actually applied). [I see you've changed the one example we have so it uses 3 values rather than 1 - that's good but I'd personally keep a 1-value example as that's probably the most common use-case, and show a 4-value one as well, and maybe a 2-value one too. Up to you though, the main point is I think it needs more than one example.]

Reply:
I'll do an example for each. That means we'll have a chromatic snapshot for each should anything go wrong in future.